### ISSUE TITLE
Is it possible to support Redis sentinel?

### ISSUE THREAD

**Issue by ChopperLee2011** [mher/node-celery#97]
I gonna using Redis sentinel for celery broker, Celery 4.0 supports this feature.
after I check Readme for this lib, don't find any description about Redis sentinel, and when I pass `sentinel://localhost:26379/0` to celeryBroker config value, it returns "Error: Unsupported broker type: sentinel". so I think it is not been supported in this lib, right? any plan or it is a won't fix feature?

**Comment by CCharlieLi** [comment]
I agree with Chopper's comment, Celery is a awesome project that could be used in enterprise development, Redis sentinel is also expected as database for broker and backend. Thus, I've made a PR( https://github.com/mher/node-celery/pull/98) to add sentinel support for node-celery. @mher  I'd like to have you review the code and any comments are welcome! Thanks!
